project-metrics/metrics_service/models.py

```diff
@@ -42,3 +42,29 @@ class MetricResult(Base):
   def __repr__(self) -> Text:
     return "<Metric(name='%s', value='%.3g', computed_at='%s')>" % (
         self.name, self.value, self.computed_at)
+
+
+class TravisState(enum.Enum):
+  """A state of a Travis build or job."""
+  CREATED = 0
+  PENDING = 1
+  PASSED = 2
+  FAILED = 3
+  ERRORED = 4
+  CANCELLED = 5
+
+
+class Build(Base):
+  """A Travis build."""
+
+  __tablename__ = 'travis_builds'
+
+  id = sqlalchemy.Column(sqlalchemy.Integer, primary_key=True)
+  number = sqlalchemy.Column(sqlalchemy.Integer)
+  duration = sqlalchemy.Column(sqlalchemy.Integer)
+  state = sqlalchemy.Column(sqlalchemy.Enum(TravisState))
+  started_at = sqlalchemy.Column(sqlalchemy.DateTime)
+
+  def __repr__(self) -> Text:
+    return "<Build(number='%d', duration='%d', state='%s', started_at='%s')>" % (
```

Comment:
Nit: none of there values are strings, so no need to surround with `' '`

Reply:
Removed.